chore(Eufemia): update Firefox to newest version 123.0 (#3392)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/contribute/faq.mdx

Jest does start several works at the same time. When we simulate states on e.g. ToggleButton and Button – Playwright (v1.31.2) struggles to handle this. Some of the hover or focus tests will fail. To ensure Jest we never execute several workers as the same time, we set `--maxWorkers=1`.

### Installing screenshot browser

> We run screenshot tests in a Firefox browser, to install this you need to run

Reviewer: ```suggestion
For running screenshot tests (visual tests), a headless Firefox browser is needed. 

In normal circumstances, the browser will be installed automatically via yarn install – but to either upgrade to a newer version, or to wipe out existing versions and run the install manually, you can run this commands, depending on your needs:
```

```

> yarn workspace @dnb/eufemia playwright install firefox

Reviewer: ```suggestion
yarn workspace @dnb/eufemia playwright uninstall --all
yarn workspace @dnb/eufemia add -D @playwright/test
yarn workspace @dnb/eufemia playwright install firefox

yarn workspace dnb-design-system-portal playwright uninstall --all
yarn workspace dnb-design-system-portal add -D @playwright/test
yarn workspace dnb-design-system-portal playwright install firefox
```

```

> 
> after `yarn install`.

Reviewer: ```suggestion
```
